### ISSUE TITLE
ELON WE LOVE U

### ISSUE THREAD

**Issue by therealbluepandabear** [twitter/the-algorithm#634]
Elon we love you don't let these wokies tell you what to do!

**Comment by guilherssousa** [comment]
no, we don't

**Comment by rainisthename** [comment]
lol

**Comment by RocketRaccoon27** [comment]
[![Watch the video](https://img.youtube.com/vi/aAvnMbl6fZE/default.jpg)](https://youtu.be/aAvnMbl6fZE)

**Comment by Sqaaakoi** [comment]
you called @RocketRaccoon27 a boomer in another issue and in this one too I think you are the boomer smh

**Comment by yagihime** [comment]
you sleep

**Comment by Sqaaakoi** [comment]
> > [![Watch the video](https://camo.githubusercontent.com/01f4c3e4bc362210435cf2e9430b1c13e5b72dc9f401c6088b9ee1ac2ff4553e/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f6141766e4d626c36665a452f64656661756c742e6a7067)](https://youtu.be/aAvnMbl6fZE)
> 
> how's mastodon going?

great without alt-right bigots

**Comment by RocketRaccoon27** [comment]
> you called @RocketRaccoon27 a boomer in another issue and in this one too I think you are the boomer smh

Bro hasn't touched grass since 2019 fr fr

**Comment by RocketRaccoon27** [comment]
> > > you called @RocketRaccoon27 a boomer in another issue and in this one too I think you are the boomer smh
> > 
> > 
> > Bro hasn't touched grass since 2019 fr fr
> 
> u mad that mastodon failed? lmao, you belong there L

Random L inserted for no reason, huh. Is Sneako not streaming?

**Comment by Sqaaakoi** [comment]
> u mad that mastodon failed? lmao, you belong there L
"mastodon" "failed"

u dont understand the difference between talking to real people and trying to farm likes and reposts

**Comment by Sqaaakoi** [comment]
> How much people are in your head rent-free, wokie?

pretty sure calling people "wokie" is a typical alt-right thing to do

**Comment by yagihime** [comment]
go to bed

**Comment by tfw7343** [comment]
I love how people can see the changes elons made, and the shit features and things he changed to the algo, but people still "love him" for "telling the wokies to fuck themselves" that they even start stupid issues like this lmao

Like cmon man, at least be funny lol

**Comment by Sqaaakoi** [comment]
> it's 4:20 PM

only you would live in the timezone +12:57

**Comment by yagihime** [comment]
up mine! narcoleptic amoralists!

**Comment by Sqaaakoi** [comment]
> 



> @Sqaaakoi this explains things:
> 
> ![image](https://user-images.githubusercontent.com/50536495/229263401-4d5a9f75-7ae9-44b6-b0cf-11b4f961359f.png)

imagine losing an argument by saying "pronouns in github readme"

**Comment by thealexparadox** [comment]
who is "we"? the asskissers are vastly outnumbered by those who think he's an arsehole.

**Comment by thealexparadox** [comment]
> You're a sick demented woketard who agrees with censorship, you going straight to HELL lmao

pfff. nice try insulting someone you don't know a thing about. try pulling your head out of Elon's ass once in a while.

**Comment by tfw7343** [comment]
> Yall are a bunch of wokies that have achieved NOTHING in your life. You are all losers.
> 
> 
> 
> You are all just mad because Elon is a billionaire while you still live in your mum's basement at 30. 
> 
> 
> 
> 

For someone who's 17 I wouldn't be saying this shit lmao, why are you coming here just to start shit?

**Comment by tfw7343** [comment]
> > > Yall are a bunch of wokies that have achieved NOTHING in your life. You are all losers.
> 
> > > You are all just mad because Elon is a billionaire while you still live in your mum's basement at 30.
> 
> > 
> 
> > For someone who's 17 I wouldn't be saying this shit lmao, why are you coming here just to start shit?
> 
> 
> 
> The thing is, I don't give half a damn about your opinion, so you can head out the back door and leave bud.
> 
> 
> 
> You people are sick creatures who actively IGNORED Twitter censorship for YEARS. 
> 
> 
> 
> Don't act as if you have the moral highground. If you call Elon 'evil', you might as well apply that label to yourself.

Don't care, you're probably a freeloader who still lives with their parents smh. Try making something for yourself. Probably don't own a car, you're not even a homeowner smh. For someone who probably works at maccas you sure talk a lot of shit. Not even old enough to drink since you're an American. You don't pay taxes and you aren't even allowed to vote. Get some life skills before calling "the wokies" insults lol.

If all you're going to do is throw Ad Hominem everywhere we can do it too, you idiot lol

**Comment by tfw7343** [comment]
> 
> 
> > > > > Yall are a bunch of wokies that have achieved NOTHING in your life. You are all losers.
> 
> > > 
> 
> > > 
> 
> > > > > You are all just mad because Elon is a billionaire while you still live in your mum's basement at 30.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > For someone who's 17 I wouldn't be saying this shit lmao, why are you coming here just to start shit?
> 
> > > 
> 
> > > 
> 
> > > The thing is, I don't give half a damn about your opinion, so you can head out the back door and leave bud.
> 
> > > You people are sick creatures who actively IGNORED Twitter censorship for YEARS.
> 
> > > Don't act as if you have the moral highground. If you call Elon 'evil', you might as well apply that label to yourself.
> 
> > 
> 
> > Don't care, you're probably a freeloader who still lives with their parents smh. Try making something for yourself. Probably don't own a car, you're not even a homeowner smh. For someone who probably works at maccas you sure talk a lot of shit. Not even old enough to drink since you're an American. You don't pay taxes and you aren't even allowed to vote. Get some life skills before calling "the wokies" insults lol.
> 
> > 
> 
> > If all you're going to do is throw Ad Hominem everywhere we can do it too, you idiot lol
> 
> 
> 
> You know nothing about me, nor who I am, so I don't really take your 'attacks' personally. I am smarter than that.
> 
> 
> 
> You see, I am a strong person, because I don't take things seriously online. I don't take things personally or to heart, this is the online world.
> 
> 
> 
> You -- on the other hand -- are a weak small man, thinking you can personally attack someone you've never met. People who resort to personal attacks over random comments online are very weak.
> 
> 
> 
> Shoo, fly. You're as relevant to me as my bathrobe.
> 
> 
> 
> 
"You have achieved nothing in life", "you're all a bunch of wokies who are going to hell", "you are all losers"
For someone who doesn't care about our opinions you sure do seem to love giving out your own.
"You know nothing about me" likewise pal lol

**Comment by tfw7343** [comment]
> > > > > > > Yall are a bunch of wokies that have achieved NOTHING in your life. You are all losers.
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > > You are all just mad because Elon is a billionaire while you still live in your mum's basement at 30.
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > For someone who's 17 I wouldn't be saying this shit lmao, why are you coming here just to start shit?
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > The thing is, I don't give half a damn about your opinion, so you can head out the back door and leave bud.
> 
> > > 
> 
> > > 
> 
> > > > > You people are sick creatures who actively IGNORED Twitter censorship for YEARS.
> 
> > > 
> 
> > > 
> 
> > > > > Don't act as if you have the moral highground. If you call Elon 'evil', you might as well apply that label to yourself.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Don't care, you're probably a freeloader who still lives with their parents smh. Try making something for yourself. Probably don't own a car, you're not even a homeowner smh. For someone who probably works at maccas you sure talk a lot of shit. Not even old enough to drink since you're an American. You don't pay taxes and you aren't even allowed to vote. Get some life skills before calling "the wokies" insults lol.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > If all you're going to do is throw Ad Hominem everywhere we can do it too, you idiot lol
> 
> > > 
> 
> > > 
> 
> > > You know nothing about me, nor who I am, so I don't really take your 'attacks' personally. I am smarter than that.
> 
> > > You see, I am a strong person, because I don't take things seriously online. I don't take things personally or to heart, this is the online world.
> 
> > > You -- on the other hand -- are a weak small man, thinking you can personally attack someone you've never met. People who resort to personal attacks over random comments online are very weak.
> 
> > > Shoo, fly. You're as relevant to me as my bathrobe.
> 
> > 
> 
> > "You have achieved nothing in life", "you're all a bunch of wokies who are going to hell", "you are all losers" For someone who doesn't care about our opinions you sure do seem to love giving out your own. "You know nothing about me" likewise pal lol
> 
> 
> 
> And we'll leave it at that, bud.
> 
> 
> 
> We are enemies, and you people will always be my enemy. I will fight to see the day when you lose.
> 
> 
> 
>  You caused 2-3 years of hell, digital censorship, and tyranny. I have no mercy upon you, nor do I care.
> 
> 

I'll have you know I graduated top of my class in the Navy Seals, and I've been involved in numerous secret raids on Al-Quaeda, and I have over 300 confirmed kills. I am trained in gorilla warfare and I'm the top sniper in the entire US armed forces. You are nothing to me but just another target. I will wipe you the fuck out with precision the likes of which has never been seen before on this Earth, mark my fucking words. You think you can get away with saying that shit to me over the Internet? Think again, fucker. If only you could have known what unholy retribution your little "clever" comment was about to bring down upon you, maybe you would have held your fucking tongue. But you couldn't, you didn't, and now you're paying the price, you goddamn idiot.

**Comment by tfw7343** [comment]
> People who resort to death threats online like @tfw7343 usually have very serious mental issues, you need help bud, arguing with a guy who's probably half your age. Get a life.

I'm 18 lol, clearly you've never heard of irony.

**Comment by Sqaaakoi** [comment]
> People who resort to death threats online like @tfw7343 usually have very serious mental issues, you need help bud, arguing with a guy who's probably half your age. Get a life.

got sucked into a conservative media algorithm at the age of 12 that is so extreme that you totally skipped the section on copypasta and went straight to the defend capitalism section

**Comment by tfw7343** [comment]
> > > People who resort to death threats online like @tfw7343 usually have very serious mental issues, you need help bud, arguing with a guy who's probably half your age. Get a life.
> 
> > 
> 
> > I'm 18 lol, clearly you've never heard of irony.
> 
> 
> 
> Exactly, you're a liar. You're a little man.
> 
> 
> 
> You're the guy in the picture.
> 
> 
> 
> Little men online pretend they are things that they are not, you pretending that you are some Navy seal is just you projecting your insecurities onto others, because you are a failure. That is your destiny.

You've literally never been on the internet before have you lmao.
Google "navy seal copypasta" you moron lol

**Comment by tfw7343** [comment]
> 'omg I owned him ahh I posted a weird navy seal meme ahh i owned him mum look lmao'
> 
> 

Are you american?
You spell "mum" the "outside of America" way lol

**Comment by tfw7343** [comment]
> > > 'omg I owned him ahh I posted a weird navy seal meme ahh i owned him mum look lmao'
> 
> > 
> 
> > Are you american? You spell "mum" the "outside of America" way lol
> 
> 
> 
> no I ain't american bud, I'm from the great commie country of New Xiland that's been run in the waters by people like you that just assaulted a 5'1 woman at Auckland a week ago lmao. 'Peaceful' my ass, yall are literally the most violent people on the face of the planet.

I'm from australia dumbass, lol
Talk about assuming stuff. Your country is literally a brother to mine. The irony of of talking about how shit New Zealand is compared to sucking the dick of capitalism so hard you forget how good it is in New Zealand lol

**Comment by tfw7343** [comment]
> > > > > 'omg I owned him ahh I posted a weird navy seal meme ahh i owned him mum look lmao'
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Are you american? You spell "mum" the "outside of America" way lol
> 
> > > 
> 
> > > 
> 
> > > no I ain't american bud, I'm from the great commie country of New Xiland that's been run in the waters by people like you that just assaulted a 5'1 woman at Auckland a week ago lmao. 'Peaceful' my ass, yall are literally the most violent people on the face of the planet.
> 
> > 
> 
> > I'm from australia dumbass, lol Talk about assuming stuff. Your country is literally a brother to mine. The irony of of talking about how shit New Zealand is compared to sucking the dick of capitalism so hard you forget how good it is in New Zealand lol
> 
> 
> 
> new zealand is becomin pretty shit though in recent years, but it's like talking to a brick wall with you lot. I can smell your love for socialism 3,000 km away just by that reply.
> 
> 
> 
> plus, you're an idiot, I knew you were from aus just by the way you spoke.
> 
> 
> 
> my point was that you are all literally the same and think the same way. anyone who thinks outside your echo chamber is an 'alt-right' person or whatever, just get out of here dude, you people are terrible and gross.

"I'm not alt right"
Casually calls an entire nation terrible, gross, who "think the same way" and that we're all socialists
Nice trolling lol

**Comment by User0110001100** [comment]
EMMMM.....

**Comment by tfw7343** [comment]
> > > > > > > 'omg I owned him ahh I posted a weird navy seal meme ahh i owned him mum look lmao'
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > Are you american? You spell "mum" the "outside of America" way lol
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > no I ain't american bud, I'm from the great commie country of New Xiland that's been run in the waters by people like you that just assaulted a 5'1 woman at Auckland a week ago lmao. 'Peaceful' my ass, yall are literally the most violent people on the face of the planet.
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > I'm from australia dumbass, lol Talk about assuming stuff. Your country is literally a brother to mine. The irony of of talking about how shit New Zealand is compared to sucking the dick of capitalism so hard you forget how good it is in New Zealand lol
> 
> > > 
> 
> > > 
> 
> > > new zealand is becomin pretty shit though in recent years, but it's like talking to a brick wall with you lot. I can smell your love for socialism 3,000 km away just by that reply.
> 
> > > plus, you're an idiot, I knew you were from aus just by the way you spoke.
> 
> > > my point was that you are all literally the same and think the same way. anyone who thinks outside your echo chamber is an 'alt-right' person or whatever, just get out of here dude, you people are terrible and gross.
> 
> > 
> 
> > "I'm not alt right" Casually calls an entire nation terrible, gross, who "think the same way" and that we're all socialists Nice trolling lol
> 
> 
> 
> gosh dude, imagine using the world 'alt-right' in 2023.
> 
> 
> 
> First of all, the 'alt-right' or whatever it is an American thing.
> 
> 
> 
> Second of all, the 'alt-right' fad became dead YEARS ago.
> 
> 
> 
> Third of all, making jokes about how terrible New Xiland has become does not make me 'alt-right'.
> 
> 
> 
> You're literally proving my point, all you do is accuse others of being fascists yet the true fascist is the person that YOU see in the mirror every day, and you know it deep down.
> 
> 
> 
> YOU are the ones who always ask for online censorship when people disagree with you.
> 
> 
> 
> YOU are the ones who assaulted a 5'1 woman at Auckland a week ago.
> 
> 
> 
> YOU are the ones who are willing to kill someone if they don't call you a 'they/them'.
> 
> 
> 
> Yall are the true far right, Yall are literally the true fascists that we were warned about for many years
> 
> 
> 
> You act all tough online, yet I've never heard anyone say it to my face, why don't you try me? 
> 
> 
> 
> Add me on Discord thebluepandabear#9011, let's have a real voice conversation and settle this.

Lmao stay mad

**Comment by yagihime** [comment]
This Issue Has Been Closed For Quite Some Time

**Comment by tfw7343** [comment]
> and guys, I called it, the guy is an inpatient weirdo who couldn't wait like 2-3 hours to have a voice conversation, as if his life depended on having it NOW.
> 
> 
> 
> good representation of modern society, lack of patience
> 
> 
> 
> be grateful yo ass would've been roasted in that voice chat

- Lmfao, I'm sorry I have a job and a life other than arguing with people online, unlike you. My shift ends at 9pm. Unblock me if you're not scared lol![image](https://user-images.githubusercontent.com/102307371/229266917-bd914d85-96e5-4d55-967e-833b4bcdaaac.jpeg)

**Comment by Sqaaakoi** [comment]
> > > > > > > 'omg I owned him ahh I posted a weird navy seal meme ahh i owned him mum look lmao'
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > > Are you american? You spell "mum" the "outside of America" way lol
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > no I ain't american bud, I'm from the great commie country of New Xiland that's been run in the waters by people like you that just assaulted a 5'1 woman at Auckland a week ago lmao. 'Peaceful' my ass, yall are literally the most violent people on the face of the planet.
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > I'm from australia dumbass, lol Talk about assuming stuff. Your country is literally a brother to mine. The irony of of talking about how shit New Zealand is compared to sucking the dick of capitalism so hard you forget how good it is in New Zealand lol
> > > 
> > > 
> > > new zealand is becomin pretty shit though in recent years, but it's like talking to a brick wall with you lot. I can smell your love for socialism 3,000 km away just by that reply.
> > > plus, you're an idiot, I knew you were from aus just by the way you spoke.
> > > my point was that you are all literally the same and think the same way. anyone who thinks outside your echo chamber is an 'alt-right' person or whatever, just get out of here dude, you people are terrible and gross.
> > 
> > 
> > "I'm not alt right" Casually calls an entire nation terrible, gross, who "think the same way" and that we're all socialists Nice trolling lol
> 
> gosh dude, imagine using the world 'alt-right' in 2023.
> 
> First of all, the 'alt-right' or whatever it is an American thing.
> 
> Second of all, the 'alt-right' fad became dead YEARS ago.
> 
> Third of all, making jokes about how terrible New Xiland has become does not make me 'alt-right'.
> 
> You're literally proving my point, all you do is accuse others of being fascists yet the true fascist is the person that YOU see in the mirror every day, and you know it deep down.
> 
> YOU are the ones who always ask for online censorship when people disagree with you.
> 
> YOU are the ones who assaulted a 5'1 woman at Auckland a week ago.
> 
> YOU are the ones who are willing to kill someone if they don't call you a 'they/them'.
> 
> Yall are the true far right, Yall are literally the true fascists that we were warned about for many years
> 
> You act all tough online, yet I've never heard anyone say it to my face, why don't you try me?
> 
> Add me on Discord thebluepandabear#9011, let's have a real voice conversation and settle this.

did you go counter protesting against that or something? if you did what happened, did you get upset by the protesters?

**Comment by Funtimes909** [comment]
> Yall are a bunch of wokies that have achieved NOTHING in your life. You are all losers.
> 
> You are all just mad because Elon is a billionaire while you still live in your mum's basement at 30.
ironic coming from the person who sits inside behind the computer voicing their political opinions on a fucking github repository

**Comment by Funtimes909** [comment]
> lmao you're a 'rust' developer from New Xiland who has their pronouns in their profile @Funtimes909 stfu you wokies are running down new zealand yall literally assaulted a 5'1 woman stfu


And what are you, some sad fucker who can't shut the fuck up when getting called out, judging from the shit you've said here you'd probably be the one standing off to the sidelines filming the assault happening instead of help, seek help you sick bastard

**Comment by Funtimes909** [comment]
> > > lmao you're a 'rust' developer from New Xiland who has their pronouns in their profile @Funtimes909 stfu you wokies are running down new zealand yall literally assaulted a 5'1 woman stfu
> 
> > 
> 
> > And what are you, some sad fucker who can't shut the fuck up when getting called out, judging from the shit you've said here you'd probably be the one standing off to the sidelines filming the assault happening instead of help, seek help you sick bastard
> 
> 
> 
> Lmao, just gtfo, New Zealand would be much better if you weirdos didn't exist here. You must be gross in real life ay?
> 
> 
> 
> You bring shame among our country.


Hey at least I come up with original insults rather then calling everyone who hurts your feelings "Wokies"

**Comment by Funtimes909** [comment]
> Behold guys, this is the woman who is telling me I'm a basement dweller, the person who spends time of their day coding a fucking cum mod. What a sad, pathetic little cunt you are. 

Hey! You called me a women! Thank you!! I'm transgender so it's nice to be gendered correctly!
And yeah I did make that, it barely works, and I'm sure a lot of people will find it a lot more appealing then the scummy shit you've said in the past 6 hours, stop sucking elons dick an touch grass for the first time in your life

**Comment by Funtimes909** [comment]
> > Behold guys, this is the woman who is telling me I'm a basement dweller, the person who spends time of their day coding a fucking cum mod. What a sad, pathetic little cunt you are. 
> 
> Hey! You called me a women! Thank you!! I'm transgender so it's nice to be gendered correctly!
> And yeah I did make that, it barely works, and I'm sure a lot of people will find it a lot more appealing then the scummy shit you've said in the past 6 hours, stop sucking elons dick an touch grass for the first time in your life 


Also how fucking dumb do you have to be to notice that the last commit was literally half a year ago, actual commit with code was much longer ago

**Comment by Sqaaakoi** [comment]
> I can only imagine this woman talking in real life:
> 
> 'ay ilon u are a buch yew auh transphobuc and ruycust' rofl

based upon this attitude i would bet money you and your friends are snapchat users who vape on school busses

**Comment by Funtimes909** [comment]
> Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are. 
> 
> 
> 
> You are all little slimey fucks.


I once again raise the point that sucking elons dick isn't going to get you anywhere, neither is being a sad loser who insults people for fun over the internet. We get it, you really have nothing better to do with your life other than treat Elon musk like some MrBeast of Twitter hoping that he'll give you a Twitter checkmark for free because you fondled his balls on the Twitter algorithm GitHub

**Comment by Funtimes909** [comment]
> > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are. 
> > 
> > 
> > 
> > You are all little slimey fucks.
> 
> 
> I once again raise the point that sucking elons dick isn't going to get you anywhere, neither is being a sad loser who insults people for fun over the internet. We get it, you really have nothing better to do with your life other than treat Elon musk like some MrBeast of Twitter hoping that he'll give you a Twitter checkmark for free because you fondled his balls on the Twitter algorithm GitHub 


You are literally the fucking dumb NPC side character from every 2004 PlayStation game

**Comment by tfw7343** [comment]
> Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are. 
> 
> 
> 
> You are all little slimey fucks.

Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)

"Delete that image plzzzzzzz it's making me look bad!!!!" 

Get a life dude. "I'm too scared my parents will hear me in the vc"

**Comment by Funtimes909** [comment]
> > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are. 
> > 
> > 
> > 
> > You are all little slimey fucks.
> 
> Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> 
> "Delete that image plzzzzzzz it's making me look bad!!!!" 
> 
> Get a life dude. "I'm too scared my parents will hear me in the vc"


This kid is literally what I imagine every nine year old on Xbox live in 2013 to be like

**Comment by yagihime** [comment]
this kid's still at this bullshit???
https://www.youtube.com/watch?v=kT2ImTVFZoY

> This kid is literally what I imagine every nine year old on Xbox live in 2013 to be like

being a late 00's-early 10's xbox live rager kid is actually a far more productive activity than being a billionare dickrider kid going to bat for the emerald baby over the april fool purporting to be precisely that which eroded thine brain. a case study in action. maybe we should re-open the issue, as it is present, and pressing. many such cases!

**Comment by yagihime** [comment]
> rent-free

no, open source

**Comment by yagihime** [comment]
> You're a fucking idiot who watches the news all day and believes what the media says.

is that what the media is telling you or what???? i wouldn't know

**Comment by tfw7343** [comment]
> > > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are.
> 
> > > You are all little slimey fucks.
> 
> > 
> 
> > Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> 
> > 
> 
> > "Delete that image plzzzzzzz it's making me look bad!!!!"
> 
> > 
> 
> > Get a life dude. "I'm too scared my parents will hear me in the vc"
> 
> 
> 
> you never 'deleted' it sir, you still kept the link on the post even though I asked you nicely to remove it since it makes me look bad.

I removed the link lol
You're the one that didn't delete the reply moron

**Comment by yagihime** [comment]
> > > You're a fucking idiot who watches the news all day and believes what the media says.
> > 
> > 
> > is that what the media is telling you or what???? i wouldn't know
> 
> I know that you wouldn't know, because you don't know shit

(speaking english) 
how do you know what i know?? how do know how i know?? it mustn't, nay, couldn't be due to the heavily curated information (media) _you_ receive ("for you") through the information superhighway regularly, determined from your personal information - including, but not limited to, lattitude, longitude, internet protocol address, interaction history (including mere views) and payment status, hoarded on a server in perpetuity. billions of boosted splurtchians for millions of withering, terminally-online brains. your data, my data, the data of billions of other humans, tucked away in noisy, hot warehouses dedicated to the sole act of hoarding data, to be used against the populace in both overt and subtle ways. all that, under the facade of being "for you". it sure as shit ain't for us. it has the very same nefarious intent as the form of media you (very incorrectly) assume i be smoothin my damn brain with. so, go ahead and believe that i, admittedly an aging hag, must spend all my day, drooling, deluded, experiencing passive boomer brainwashing, because of my five or so sentences on this shitshow april fools joke repo? if you are so quick to believe that, so eager to ride the dick of some ghoul who would gladly step on us all like ants, then who's believing what the media (which is under their control) says to them? 

if you are having difficulty with your reading comprehension beyond a paragraph: smdh, this twitter shit, nah fuck it, all that social media shit making you hella stupid lil homie. get out before it's too late, the ghouls and the feds worldwide using you as a pawn in a zero-sum game. go micspam a counter-strike server or something, you'll have more fun, trust me.

**Comment by Funtimes909** [comment]
> Bud, who even are you? are you angry at Musk for releasing the source code and promoting transparency? What are even your 'points'? You're a fucking idiot who watches the news all day and believes what the media says. I'd knock you out in a heartbeat.
> 
> also get tf away from me, freak. I would rather die than be near people like you vomiting_face at least the other people are half interesting to talk to, you're just an old creepy hag
> 
> So much smelly woketards in this thread vomiting_face vomiting_face


Earlier in the thread you stated that death threats don't get you anywhere, and here you are threatening violence against someone who hurt your feelings. I'd say watching the news is probably a better hobby then doing the shit that you're doing, you act like every entitled bitch karen at the mcdonalds when their order is 5 minutes late, stop acting like the world revolves around you. You're probably the one person from the family reunion that can't go because "wearing face masks cause autism" I mean maybe that would explain a lot about you. stop riding the dicks of right winged bigots you daft sack of shit

**Comment by yagihime** [comment]
> Who tf asked? You're literally the least funny person in this thread, at least @Funtimes909 is interesting stuck_out_tongue
> 
> This is my thread.

oh hun i'm not trying to be funny, don't worry.

you opened an issue for software, not a thread. it belongs to all, including those who can view and access this repository, with or without an account. this aint no fuckin somethingawful. an issue is opened for the many users of github, not your vanity. it has long since been resolved AND closed by a team member. 

statistically speaking, over the overwhelming majority of the posts in in this long-closed issue are _you_, complaining about how asleep you are or wish to be, largely because thats what the really rich user/most divorced man on the bird app told you to think. there is also how anyone over the age of 25 who has completed a high school education was born between the years of 1946 and 1964 or something. quite a contribution.

**Comment by Funtimes909** [comment]
> Yo funtimes wanna debate on discord?


I only add people who I think are nice, you're just some fucking NPC who goes out of their way to make others feel bad, log off and get some bitches you brainless bigot

**Comment by Funtimes909** [comment]
> > > > > You're a fucking idiot who watches the news all day and believes what the media says.
> > > > 
> > > > 
> > > > is that what the media is telling you or what???? i wouldn't know
> > > 
> > > 
> > > I know that you wouldn't know, because you don't know shit
> > 
> > 
> > (speaking english) how do you know what i know?? how do know how i know?? it mustn't, nay, couldn't be due to the heavily curated information (media) _you_ receive ("for you") through the information superhighway regularly, determined from your personal information - including, but not limited to, lattitude, longitude, internet protocol address, interaction history (including mere views) and payment status, hoarded on a server in perpetuity. billions of boosted splurtchians for millions of withering, terminally-online brains. your data, my data, the data of billions of other humans, tucked away in noisy, hot warehouses dedicated to the sole act of hoarding data, to be used against the populace in both overt and subtle ways. all that, under the facade of being "for you". it sure as shit ain't for us. it has the very same nefarious intent as the form of media you (very incorrectly) assume i be smoothin my damn brain with. so, go ahead and believe that i, admittedly an aging hag, must spend all my day, drooling, deluded, experiencing passive boomer brainwashing, because of my five or so sentences on this shitshow april fools joke repo? if you are so quick to believe that, so eager to ride the dick of some ghoul who would gladly step on us all like ants, then who's believing what the media (which is under their control) says to them?
> > if you are having difficulty with your reading comprehension beyond a paragraph: smdh, this twitter shit, nah fuck it, all that social media shit making you hella stupid lil homie. get out before it's too late, the ghouls and the feds worldwide using you as a pawn in a zero-sum game. go micspam a counter-strike server or something, you'll have more fun, trust me.
> 
> Who tf asked? You're literally the least funny person in this thread, at least @Funtimes909 is interesting stuck_out_tongue
> 
> This is my thread.


Narcissistic personality disorder is a mental health condition in which people have an unreasonably high sense of their own importance. They need and seek too much attention and want people to admire them. People with this disorder may lack the ability to understand or care about the feelings of others.

**Comment by Oman395** [comment]
This is genuinely one of the most entertaining threads I've ever gone through in a while. Thanks @therealbluepandabear for being dumb enough to continue attempting to argue against a brick wall (made up of the vast majority of people), if you want to debate someone my discord is AutisticMOFO#0521

**Comment by Funtimes909** [comment]
> > > Who tf asked? You're literally the least funny person in this thread, at least @Funtimes909 is interesting stuck_out_tongue
> > > This is my thread.
> > 
> > 
> > oh hun i'm not trying to be funny, don't worry.
> > you opened an issue for software, not a thread. it belongs to all, including those who can view and access this repository, with or without an account. this aint no fuckin somethingawful. an issue is opened for the many users of github, not your vanity. it has long since been resolved AND closed by a team member.
> > statistically speaking, over 50% of the posts in in this long-closed issue are _you_, complaining about how asleep you are or wish to be, largely because thats what the really rich user/most divorced man on the bird app told you to think. there is also how anyone over the age of 25 who has completed a high school education was born between the years of 1946 and 1964 or something. quite a contribution.
> > *edit: and the hextuple posting!
> 
> ![IMG_20230402_151807](https://user-images.githubusercontent.com/50536495/229329260-cac167bb-b7e4-457e-9e1c-095b66e72b27.jpg)

https://user-images.githubusercontent.com/68979259/229329443-22b3f952-822f-44df-a987-8b9d5b4f6f80.mp4

**Comment by Funtimes909** [comment]
> > 
> 
> Trust me, y'all ain't the vast majority of people.


So far we thankfully haven't seen anyone else dumb enough to side with an opinion as messed up as yours

**Comment by Funtimes909** [comment]
> ![image](https://user-images.githubusercontent.com/50536495/229284709-1513d290-84fa-4261-b3bc-d9e76af1f3a9.png)
> 
> look at this woke retarded feminist

Only saw this now, My profile is a lot more populated than yours bro, why you have your activity private? Scared a future employer is gonna see that you post transphobic and right winged bigoted shit on a github repository and immediately turn you down? or just down right embarrassed by the shit you pull

**Comment by Funtimes909** [comment]
> Anyone else brave enough to debate me?
> 
> @Funtimes909 I think we should debate.


Whats the point. Do you get off to arguing with people or something. It's not like I can change your mind on anything

**Comment by Funtimes909** [comment]
> > Anyone else brave enough to debate me?
> > @Funtimes909 I think we should debate.
> 
> Whats the point. Do you get off to arguing with people or something. It's not like I can change your mind on anything


Stop acting like we're all pussies because we don't want to "debate you"
Fucks sake probably because no one wants to fucking waste their time on you

**Comment by Funtimes909** [comment]
> > > Anyone else brave enough to debate me?
> > > @Funtimes909 I think we should debate.
> > 
> > 
> > Whats the point. Do you get off to arguing with people or something.
> 
> Geez. Cum mod and now this. Get off PH for once roll_eyes
> 
> > It's not like I can change your mind on anything
> 
> i actually changed my mind on some topics after debating the other guy


Like what? Calling trans people slurs? Calling yourself a neo nazi? Telling people to kill themselves because their jews?
yeah thats right my friend told me the shit you said in vc

**Comment by Funtimes909** [comment]
> > > > Anyone else brave enough to debate me?
> > > > @Funtimes909 I think we should debate.
> > > 
> > > 
> > > Whats the point. Do you get off to arguing with people or something. It's not like I can change your mind on anything
> > 
> > 
> > Stop acting like we're all pussies because we don't want to "debate you" Fucks sake probably because no one wants to fucking waste their time on you
> 
> > > > > Anyone else brave enough to debate me?
> > > > > @Funtimes909 I think we should debate.
> > > > 
> > > > 
> > > > Whats the point. Do you get off to arguing with people or something.
> > > 
> > > 
> > > Geez. Cum mod and now this. Get off PH for once roll_eyes
> > > > It's not like I can change your mind on anything
> > > 
> > > 
> > > i actually changed my mind on some topics after debating the other guy
> > 
> > 
> > Like what? Calling trans people slurs? Calling yourself a neo nazi? Telling people to kill themselves because their jews? yeah thats right my friend told me the shit you said in vc
> 
> Lies.


![image](https://user-images.githubusercontent.com/68979259/229343759-f3f99678-c4a5-44b8-8bc4-3806cf18d5ce.png)
Cope

**Comment by Funtimes909** [comment]
> > > > > > Anyone else brave enough to debate me?
> > > > > > @Funtimes909 I think we should debate.
> > > > > 
> > > > > 
> > > > > Whats the point. Do you get off to arguing with people or something. It's not like I can change your mind on anything
> > > > 
> > > > 
> > > > Stop acting like we're all pussies because we don't want to "debate you" Fucks sake probably because no one wants to fucking waste their time on you
> > > 
> > > 
> > > > > > > Anyone else brave enough to debate me?
> > > > > > > @Funtimes909 I think we should debate.
> > > > > > 
> > > > > > 
> > > > > > Whats the point. Do you get off to arguing with people or something.
> > > > > 
> > > > > 
> > > > > Geez. Cum mod and now this. Get off PH for once roll_eyes
> > > > > > It's not like I can change your mind on anything
> > > > > 
> > > > > 
> > > > > i actually changed my mind on some topics after debating the other guy
> > > > 
> > > > 
> > > > Like what? Calling trans people slurs? Calling yourself a neo nazi? Telling people to kill themselves because their jews? yeah thats right my friend told me the shit you said in vc
> > > 
> > > 
> > > Lies.
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/68979259/229343759-f3f99678-c4a5-44b8-8bc4-3806cf18d5ce.png) Cope
> 
> Boohoo your friend asked for it by not being serious.


So you admit you said all that?

**Comment by Funtimes909** [comment]
> Ofc i wouldnt say that, the discord acc is fake.


You mean the discord account with the exact same username as your github and the exact same one there are screenshots above of interacting with another user in this tread? LMFAO

**Comment by Funtimes909** [comment]
> > Ofc i wouldnt say that, the discord acc is fake.
> 
> You mean the discord account with the exact same username as your github and the exact same one there are screenshots above of interacting with another user in this tread? LMFAO


![image](https://user-images.githubusercontent.com/68979259/229344378-bcdd8328-5dea-41c8-ba49-0a8ddb04d9fb.png)
You mean this one?

**Comment by Funtimes909** [comment]
> > > Ofc i wouldnt say that, the discord acc is fake.
> > 
> > 
> > You mean the discord account with the exact same username as your github and the exact same one there are screenshots above of interacting with another user in this tread? LMFAO
> 
> L


Cope

**Comment by Funtimes909** [comment]
> 

![image](https://user-images.githubusercontent.com/68979259/229344569-e47289fd-20db-41cc-8a06-dfa549ee7953.png)

Did I ever say I was trying to get you banned

**Comment by Funtimes909** [comment]
> 



> > > 
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/68979259/229344569-e47289fd-20db-41cc-8a06-dfa549ee7953.png)
> > Did I ever say I was trying to get you banned
> 
> you think im intimidated by you?


Would you like me to post snippets of the video, including parts where you say racist and homophobic stuff over voice, are you seriously dumb enough to think people will believe you when you say that screenshot isn't of you
LMFAO, You are the reason new zealand isn't in the top 10 average IQ

**Comment by Funtimes909** [comment]
> > > 
> > 
> > 
> > > > > 
> > > > 
> > > > 
> > > > ![image](https://user-images.githubusercontent.com/68979259/229344569-e47289fd-20db-41cc-8a06-dfa549ee7953.png)
> > > > Did I ever say I was trying to get you banned
> > > 
> > > 
> > > you think im intimidated by you?
> > 
> > 
> > Would you like me to post snippets of the video, including parts where you say racist and homophobic stuff over voice, are you seriously dumb enough to think people will believe you when you say that screenshot isn't of you LMFAO, You are the reason new zealand isn't in the top 10 average IQ
> 
> post it, it aint me rofl


Alright bro, I'll get back to you in the morning about that, as it is getting rather late, and I am not about to open my video editor to edit a compilation of racist github user at quarter to 10.

**Comment by Funtimes909** [comment]
> > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are.
> > You are all little slimey fucks.
> 
> Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> 
> "Delete that image plzzzzzzz it's making me look bad!!!!"
> 
> Get a life dude. "I'm too scared my parents will hear me in the vc"


So [therealbluepandabear](https://github.com/therealbluepandabear) whos this then

**Comment by Sqaaakoi** [comment]
# **breaking news**
**transphobe who cried about pronouns in github readme is a neo nazi, nobody is surprised**

**Comment by Funtimes909** [comment]
> > > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are.
> > > You are all little slimey fucks.
> > 
> > 
> > Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> > "Delete that image plzzzzzzz it's making me look bad!!!!"
> > Get a life dude. "I'm too scared my parents will hear me in the vc"
> 
> So [therealbluepandabear](https://github.com/therealbluepandabear) whos this then


Cringe dumbass gets so angry that he has to go and edit his messages to make sure the person knows their blocked. Imagine being so alone in real life that you resort to being a downright idiot to others online to jack off to their reactions

**Comment by yagihime** [comment]
we have reached a milestone: prior to this very message, 49.67% of the cringe in this issue discussion comes from a single source. what's the issue again?

**Comment by Funtimes909** [comment]
> 


So are you going to give any proof it isn't actually you or wait for me to post evidence that it actually is you

**Comment by Funtimes909** [comment]
> > > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are.
> > > You are all little slimey fucks.
> > 
> > 
> > Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> > "Delete that image plzzzzzzz it's making me look bad!!!!"
> > Get a life dude. "I'm too scared my parents will hear me in the vc"
> 
> So [therealbluepandabear](https://github.com/therealbluepandabear) whos this then


Hey bro you never answered this

**Comment by Funtimes909** [comment]
LMFAO

**Comment by yagihime** [comment]
however, it also must be noted that 17.11% of that 49.67% of cringe was prior to to the closure of this very issue. as it currently stands, the quantity of cringe is rapidly increasing.

**Comment by Sqaaakoi** [comment]
quick question, can you define what "woke" means?

**Comment by tfw7343** [comment]
> > > > Ay Elon these are the people critiquing you, people who spend their day making cum mods, people who could only wish to be 1% of successful as you are.
> 
> > > > You are all little slimey fucks.
> 
> > > 
> 
> > > 
> 
> > > Dude, I deleted the previous image cuz I respect peoples stuff, but seriously dude, you're annoying. What happened to "What I said I probably shouldn't of". ![image](https://user-images.githubusercontent.com/102307371/229286357-4e5e28cb-84df-414d-9f36-5136e4a947db.jpeg)
> 
> > > "Delete that image plzzzzzzz it's making me look bad!!!!"
> 
> > > Get a life dude. "I'm too scared my parents will hear me in the vc"
> 
> > 
> 
> > So [therealbluepandabear](https://github.com/therealbluepandabear) whos this then
> 
> 
> 
> 
> 
> Hey bro you never answered this 

He also said this lol ![image](https://user-images.githubusercontent.com/102307371/229345803-0233a810-e645-49f7-aed0-c15ccd15a415.jpeg)![image](https://user-images.githubusercontent.com/102307371/229345857-79f8ae30-8f7c-47e7-9997-b49ead136e81.jpeg)

**Comment by Funtimes909** [comment]
LMAOOOOOOOOOOOOOOOOOO

**Comment by Sqaaakoi** [comment]
no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues

**Comment by Sqaaakoi** [comment]
https://user-images.githubusercontent.com/37475903/229346066-1cb6514f-9d34-406a-b76d-9afa13f5feca.mov

**Comment by Funtimes909** [comment]
> Btw @Funtimes909 stop calling me 'bro' **dude**


Imagine changing your github username after getting called out for hate speech and death threats LMAOOOO

**Comment by Sqaaakoi** [comment]
> > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> 
> and? Ur still not they/them

ok then what are my pronouns, and what are yours

**Comment by Funtimes909** [comment]
> > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> 
> and? Ur still not they/them


I mean to me their they/them just like to them I'm she/her
So whats it to you

**Comment by Funtimes909** [comment]
> > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > 
> > > 
> > > and? Ur still not they/them
> > 
> > 
> > I mean to me their they/them just like to them I'm she/her So whats it to you
> 
> you are either a man or woman.
> 
> Cope


You forgot the third gender, imbecile, thats you

**Comment by yagihime** [comment]
> i never did

???????
![the real blue panda bear](https://user-images.githubusercontent.com/126365958/229346209-2729c646-552d-47a0-adc0-fe3448af36f3.png)

**Comment by tfw7343** [comment]
> > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> 
> > > 
> 
> > > 
> 
> > > and? Ur still not they/them
> 
> > 
> 
> > I mean to me their they/them just like to them I'm she/her So whats it to you
> 
> 
> 
> you are either a man or woman.
> 
> 
> 
> Cope

And you're an idiot, and a child

**Comment by Funtimes909** [comment]
You really think that changing your github username changes anything, you've been reported by like 5 different people in this repository, and I'm pretty sure that you were reported on discord as well
Total lack of skill on your part you dumb fucking loser

**Comment by Funtimes909** [comment]
> > > i never did
> > 
> > 
> > ??????? ![the real blue panda bear](https://user-images.githubusercontent.com/126365958/229346209-2729c646-552d-47a0-adc0-fe3448af36f3.png)
> 
> inspect element go brr


How come all the repositories from both accounts are the same, how come the profile pictures the same, must just be a coincidence that as soon as someone leaves, not 3 minutes later a guy with the same profile picture comes back and starts preaching the exact same bullshit as the first guy

**Comment by Sqaaakoi** [comment]
> > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > 
> > > 
> > > and? Ur still not they/them
> > 
> > 
> > I mean to me their they/them just like to them I'm she/her So whats it to you
> 
> you are either a man or woman.
> 
> Cope

_intersex people?_

**Comment by Funtimes909** [comment]
> > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > > 
> > > > 
> > > > and? Ur still not they/them
> > > 
> > > 
> > > I mean to me their they/them just like to them I'm she/her So whats it to you
> > 
> > 
> > you are either a man or woman.
> > Cope
> 
> _intersex people?_


Gonna go on a huge rant about how intersex are also fake, just like how the holocaust was fake according to you

**Comment by Sqaaakoi** [comment]
> > > > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > > > > 
> > > > > > 
> > > > > > and? Ur still not they/them
> > > > > 
> > > > > 
> > > > > I mean to me their they/them just like to them I'm she/her So whats it to you
> > > > 
> > > > 
> > > > you are either a man or woman.
> > > > Cope
> > > 
> > > 
> > > _intersex people?_
> > 
> > 
> > Gonna go on a huge rant about how intersex are also fake, just like how the holocaust was fake according to you
> 
> the Holocaust was real

avoiding the question?

**Comment by Sqaaakoi** [comment]
> > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > > 
> > > > 
> > > > and? Ur still not they/them
> > > 
> > > 
> > > I mean to me their they/them just like to them I'm she/her So whats it to you
> > 
> > 
> > you are either a man or woman.
> > Cope
> 
> _intersex people?_

oh you know, how I was hinting about you know, people who don't actually biologically fall into those two categories and your thoughts on them

**Comment by Sqaaakoi** [comment]
> > > > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > > > > > 
> > > > > > 
> > > > > > and? Ur still not they/them
> > > > > 
> > > > > 
> > > > > I mean to me their they/them just like to them I'm she/her So whats it to you
> > > > 
> > > > 
> > > > you are either a man or woman.
> > > > Cope
> > > 
> > > 
> > > _intersex people?_
> > 
> > 
> > oh you know, how I was hinting about you know, people who don't actually biologically fall into those two categories and your thoughts on them
> 
> i think that idc as long as they aint pushing it on me

do you honestly think biology is an agenda being pushed on you

**Comment by Funtimes909** [comment]
> > > > > > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> 
> > > > > > > > 
> 
> > > > > > > > 
> 
> > > > > > > > and? Ur still not they/them
> 
> > > > > > > 
> 
> > > > > > > 
> 
> > > > > > > I mean to me their they/them just like to them I'm she/her So whats it to you
> 
> > > > > > 
> 
> > > > > > 
> 
> > > > > > you are either a man or woman.
> 
> > > > > > Cope
> 
> > > > > 
> 
> > > > > 
> 
> > > > > _intersex people?_
> 
> > > > 
> 
> > > > 
> 
> > > > oh you know, how I was hinting about you know, people who don't actually biologically fall into those two categories and your thoughts on them
> 
> > > 
> 
> > > 
> 
> > > i think that idc as long as they aint pushing it on me
> 
> > 
> 
> > do you honestly think biology is an agenda being pushed on you
> 
> 
> 
> i think that idc as long as they aint pushing it on me.
> 
> 
> 
> End of story. If you want further debate add me on discord


This guy is so mentally unstable that he resorts to "debate me on discord" when losing an argument.
Literally the SNEAKO of GitHub except 3 times worse

**Comment by Funtimes909** [comment]
> > > > > > > > > > no its just a bit funny to do a little :trollface: to people who post hate and bigotry in github issues
> > 
> > > > > > > > > 
> > 
> > > > > > > > > 
> > 
> > > > > > > > > and? Ur still not they/them
> > 
> > > > > > > > 
> > 
> > > > > > > > 
> > 
> > > > > > > > I mean to me their they/them just like to them I'm she/her So whats it to you
> > 
> > > > > > > 
> > 
> > > > > > > 
> > 
> > > > > > > you are either a man or woman.
> > 
> > > > > > > Cope
> > 
> > > > > > 
> > 
> > > > > > 
> > 
> > > > > > _intersex people?_
> > 
> > > > > 
> > 
> > > > > 
> > 
> > > > > oh you know, how I was hinting about you know, people who don't actually biologically fall into those two categories and your thoughts on them
> > 
> > > > 
> > 
> > > > 
> > 
> > > > i think that idc as long as they aint pushing it on me
> > 
> > > 
> > 
> > > do you honestly think biology is an agenda being pushed on you
> > 
> > 
> > 
> > i think that idc as long as they aint pushing it on me.
> > 
> > 
> > 
> > End of story. If you want further debate add me on discord
> 
> 
> This guy is so mentally unstable that he resorts to "debate me on discord" when losing an argument.
> Literally the SNEAKO of GitHub except 3 times worse 


Add sneako to the list of people who's dicks you cant stop riding
But sure you can go tell all of us that we're the ones with mental illnesses and brainwashed by the media, tell us that we are the idiots who are all "woke"
Get a life Xbox kid

**Comment by yagihime** [comment]
"Then [Yeshua] comes to the talmidim and finds them sleeping, and he says to Kefa, So were you not strong enough to be awake for one hour with me?" 

Matthew 26:40

![no inspect element required](https://user-images.githubusercontent.com/126365958/229347488-d1dd591e-6021-4ecc-9d04-07f073b9bb35.png)
one more for archival's sake, just in case this kid ends up trying to coerce site admins into a debate or someshit. only tools needed are the scroll bar, spectacle and not hitting that f5 key. peace

**Comment by Funtimes909** [comment]
> > "Then [Yeshua] comes to the talmidim and finds them sleeping, and he says to Kefa, So were you not strong enough to be awake for one hour with me?"
> 
> > 
> 
> > Matthew 26:40
> 
> > 
> 
> > ![no inspect element required](https://user-images.githubusercontent.com/126365958/229347488-d1dd591e-6021-4ecc-9d04-07f073b9bb35.png) one more for archival's sake, just in case this kid ends up trying to coerce site admins into a discord debate or someshit. only tools needed are the scroll bar, spectacle and not hitting that f5 key. peace
> 
> 
> 
> im jewish dont u dare quote the NT


If you're Jewish why you call yourself a neo nazi on discord dumb fuck

**Comment by yagihime** [comment]
> im jewish dont u dare quote the NT

if that's the case you should have been avoiding lashon hara from the start!

**Comment by Funtimes909** [comment]
> > > > "Then [Yeshua] comes to the talmidim and finds them sleeping, and he says to Kefa, So were you not strong enough to be awake for one hour with me?"
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Matthew 26:40
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > ![no inspect element required](https://user-images.githubusercontent.com/126365958/229347488-d1dd591e-6021-4ecc-9d04-07f073b9bb35.png) one more for archival's sake, just in case this kid ends up trying to coerce site admins into a discord debate or someshit. only tools needed are the scroll bar, spectacle and not hitting that f5 key. peace
> 
> > > 
> 
> > > 
> 
> > > im jewish dont u dare quote the NT
> 
> > 
> 
> > If you're Jewish why you call yourself a neo nazi on discord dumb fuck
> 
> 
> 
> It was to prove my point that you are all a bunch of idiots and believe anything says online. Retard. No, I ain't no neo-nazi :rofl: 
> 
> 
> 
> > > > "Then [Yeshua] comes to the talmidim and finds them sleeping, and he says to Kefa, So were you not strong enough to be awake for one hour with me?"
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Matthew 26:40
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > ![no inspect element required](https://user-images.githubusercontent.com/126365958/229347488-d1dd591e-6021-4ecc-9d04-07f073b9bb35.png) one more for archival's sake, just in case this kid ends up trying to coerce site admins into a discord debate or someshit. only tools needed are the scroll bar, spectacle and not hitting that f5 key. peace
> 
> > > 
> 
> > > 
> 
> > > im jewish dont u dare quote the NT
> 
> > 
> 
> > If you're Jewish why you call yourself a neo nazi on discord dumb fuck
> 
> 
> 
> I'll have you know, misses 'know it all' that I am in fact not a real 'neo-nazi' nor have any far-right beliefs. My saying that on Discord was just me trolling you because you are all a bunch of gullible bafoons who believe anything that anyone says, so I played your little 'game'.


Holy shit he finally admits it was him on discord

**Comment by Funtimes909** [comment]
Imagine being so scared of getting banned off github that you change your name LMFAOOO
you say you don't care but its pretty obvious you dobecause you changed your name LOL
Anyway heres some clips of you saying you'll kill gay people, I'll make sure to send it to both discord support and github support, enjoy your ban

https://user-images.githubusercontent.com/68979259/229378422-73e9917b-f61d-4301-b74a-3604603ee596.mp4

https://user-images.githubusercontent.com/68979259/229378473-ae253a51-d893-49bf-9102-954d74ecdd00.mp4


oh and also if you say this isn't your discord account whats your real discord then? Whats your REAL discord account, and whos that in the screenshots above with @tfw7343

**Comment by Funtimes909** [comment]
Cope

**Comment by Funtimes909** [comment]
> 'ohh uhh you just admitted that you did it'
> 
> 'so you DID do it'
> 
> 'I can show them all what u did'
> 
> 'ah ah well you said you were a n-zi omg'
> 
> just shut up, yall are goofy acting like I killed someone or something? LMAO, is this a murder trial?


Its not a murder trial, but its still pretty bad telling transpeople to kill themselves

**Comment by Funtimes909** [comment]
> > Imagine being so scared of getting banned off github that you change your name LMFAOOO you say you don't care but its pretty obvious you dobecause you changed your name LOL Anyway heres some clips of you saying you'll kill gay people, I'll make sure to send it to both discord support and github support, enjoy your ban
> > oh and also if you say this isn't your discord account whats your real discord then? Whats your REAL discord account, and whos that in the screenshots above with @tfw7343
> 
> lmao that ain't even my voice, I have a kiwi accent. you're pathetic.


Post a video of you speaking here then, go on, oh are you the one whos scared now? I dare you to say "I love promoting political garbage over the internet" into your mic, and I know of software that lets me detect if its a voice changer/impersonation

**Comment by Funtimes909** [comment]
Stop trying to lie you are literally just making a fucking fool of yourself

**Comment by Funtimes909** [comment]
LMAO Got called out, failed to provide evidence, tries to run, least fucking stupid github user, stop trying to run from being banned

**Comment by Funtimes909** [comment]
"Lets act like grown ups" hes says after spreading hate online

**Comment by Sqaaakoi** [comment]
> one more for archival's sake, just in case this kid ends up trying to coerce site admins into a discord debate or someshit. only tools needed are the scroll bar, spectacle and not hitting that f5 key. peace

you don't even need the scroll bar just use your scroll wheel! maybe even download a little bit of the page if this bigoted :nerd_face: returns

**Comment by Funtimes909** [comment]
LMAO He deleted EVERYTHING, What a fucking pussy, and he calls all of us scared, look whos the fucking pissbaby now, good riddance

**Comment by tfw7343** [comment]
> LMAO He deleted EVERYTHING, What a fucking pussy, and he calls all of us scared, look whos the fucking pissbaby now, good riddance 

Lmfao, dude is coward and an idiot lol

**Comment by Sqaaakoi** [comment]
Coped very hard

**Comment by Funtimes909** [comment]
lol, pretty sure he'll still get banned anyway, hes been reported so many times with so much evidence against him, ironic that we're all "woke pussies" because we don't wanna waste our time talking to a brick wall in a "debate" on discord
Yet as soon as he realizes hes actually going to get banned, he dips, anyway unless he comes back this will be my final message, been nice talking with yall, have a nice day :)

**Comment by Oman395** [comment]
Yup, have a nice day y'all, I may or may not have 2 hours of footage deconstructing pretty much his entire argument (and getting him to accept trans people :D)

We do a little trolling (getting bigoted assholes to admit trans people exist)

**Comment by yagihime** [comment]
![FoSCcWXWcAAcwrq](https://user-images.githubusercontent.com/126365958/229532197-512ada5c-799e-4d3f-bf05-7ff1a241f6a3.png)

**Comment by Oman395** [comment]
> > Yup, have a nice day y'all, I may or may not have 2 hours of footage deconstructing pretty much his entire argument (and getting him to accept trans people :D)
> 
> > We do a little trolling (getting bigoted assholes to admit trans people exist)
> 
> Creepy that you (may) recorded it, wtf? Who is that obsessive? You're a fucking corporate bootlicker crummy faggot bud. I appreciate how you held yourself during the debate but I despise you and what you stand for.
> 
> Its cute how you all think you've owned me or whatever raised_eyebrow
> 
> You still gonna claim 20k ppl are needed to maintain Twitter, weirdo?
> 
> In the conversation I never accepted them rather the fact that they are mentally insane. Two different things.
> 
> Also I have the right to not like certain groups of people. Why force everyone to love everyone? There are certain people I don't like, so you are right if that's what you mean by the word bigoted lol
> 
> If you have an issue with that, go cry me a river.
> 
> The fact that y'all are so obsessed with me is nauseating
For one, unless you are therealbluepandabears alt, I haven't talked to you yet pretty much at all. Nobody is "obsessed" with you, it's really just fun to talk to people like you-- if anything, just to challenge my own beliefs (I've flip flopped _very_ hard on many of my previously held beliefs). Anyway, let me break down the image you sent.
1. If by "police your speech" you mean "get mad if you antagonize", then yes. Same reason you can't do things like say the n word.
2. Most women don't really give a shit about trans people using women's bathrooms-- and most trans people go out of their way to make sure people aren't uncomfortable. If you can figure out that a trans woman is trans, ask yourself why you're looking at their crotch in the bathroom.
3. ???
4. Again, ???
5. Very few trans women compete in women's sports, contrary to what the news might lead you to believe. Those who do are definitely an issue, because they have a definitive medical advantage, but that's a minority, and isn't a problem with trans people as a whole.
6. That's literally the term for it???
7. I have never seen this term used, what the fuck does it even mean?
8. See above
9. Extremely rarely used in real life
10. See above
11. See above
12. This is simply not true
13. Biological sex is assigned at birth, gender is observed.
14. Forcing kids not to harass trans people isn't really a problem. It's literally just bullying.
15. ??? Who has ever pressured anyone to put pronouns in bio?
16. No, the claim is that _trans women_ can have penises and _trans men_ can get pregnant, which is true.
17. That is a contradiction in terms and nobody actually believes that.
18. Source?
19. No, it teaches kids that biological sex is separate from mental gender
20. This is a problem, but is also relatively rare, and is also not a trans issue
21. See above
22. This is like complaining about teaching children that gay people exist-- it's just a [fact of biology](https://en.wikipedia.org/wiki/Causes_of_gender_incongruence),, so why hide it?
23. When does this happen?
24. It's child abuse to withhold necessary medical help
25. Talk to any trans person and they will tell you how hard it is to get the hormones-- hormones which are literally just the treatment for their condition
26. Again, this is treatment for gender dysphoria.
27. If you don't want to be called a bigot/transphobe, _don't be a bigot/transphobe._
28. It's really not hard to just be respectful. You aren't getting fired because of your political views, you're getting fired for being an asshole.
29. See 24

Additionally, I just want to address "rather the fact that they are mentally insane". For one, no, they aren't _insnae_-- but yes, it is a recognized mental issue. You want to know how you can treat it? _HRT, surgery, and changing gender._ That's why all of that exists. Think about it this way; could you be forced into being the opposite gender? Could someone convert you into something you aren't? The same thing applies to trans people; you can't just make them not be trans, so the only treatment option available is to make it as comfortable as possible for them.

**Comment by yagihime** [comment]
![FnejW37XkAAesLa](https://user-images.githubusercontent.com/126365958/229541791-3c99ecad-f874-4b4f-98ba-d8af0eaeef3a.jpg)

**Comment by Funtimes909** [comment]
> Oops, talking about the truth about the %&#@@&# might get me banned off GH!
> 
> Y'all are a PROTECTED CLASS lmao
> 
> Privilege.


Care to share why you purged every message before this then?
Also Imagine hiding behind the opinions of a transphobic pedophile to hide yourself from the shame your bringing upon yourself here https://www.youtube.com/watch?v=M9NApuQ8ekE

**Comment by Funtimes909** [comment]
Imagine getting so mad you go on an autistic shitspeak on github about other people enjoying their lives, just take a moment and think how fucked up your own life is before trying to ruin others

**Comment by Oman395** [comment]
> > Oops, talking about the truth about the %&#@@&# might get me banned off GH!
> > 
> > Y'all are a PROTECTED CLASS lmao
> > 
> > Privilege.
> 
> 
> Care to share why you purged every message before this then?
> Also Imagine hiding behind the opinions of a transphobic pedophile to hide yourself from the shame your bringing upon yourself here https://www.youtube.com/watch?v=M9NApuQ8ekE

I thought it was panda who cleared his messages

**Comment by Funtimes909** [comment]
> > > Oops, talking about the truth about the %&#@@&# might get me banned off GH!
> > > Y'all are a PROTECTED CLASS lmao
> > > Privilege.
> > 
> > 
> > Care to share why you purged every message before this then?
> > Also Imagine hiding behind the opinions of a transphobic pedophile to hide yourself from the shame your bringing upon yourself here https://www.youtube.com/watch?v=M9NApuQ8ekE
> 
> I thought it was panda who cleared his messages


Yeah, he did

**Comment by logicP** [comment]
"Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem

"You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath

"You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor

May I go on?

**Comment by yagihime** [comment]
> You only hate Musket because the news told you to.



< < implying i give a hoot about any news

**Comment by logicP** [comment]
> > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > May I go on?
> 
> Ay man go fuck yourself alright?



> > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > May I go on?
> 
> Elon Musk is currently sitting very cosily in your small little rat brain, and he's not paying you any rent too.
Says the same person who's absolutely obsessed with being a troll and getting pissed off at someone calling you out.

Also, you fuck yourself you transphobic little tool. You have absolutely no right to claim to be holier than thou on who or what is pedophilia when y'all can't even get a hold of your own pedophilia problem. So shut the hell up and sit down

**Comment by yagihime** [comment]
@jdm-twtr

**Comment by logicP** [comment]
> > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > May I go on?
> > > 
> > > 
> > > Ay man go fuck yourself alright?
> > 
> > 
> > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > May I go on?
> > > 
> > > 
> > > Elon Musk is currently sitting very cosily in your small little rat brain, and he's not paying you any rent too.
> > 
> > 
> > Says the same person who's absolutely obsessed with being a troll and getting pissed off at someone calling you out. Also, you fuck yourself you transphobic little tool. You have absolutely no right to claim to be holier than thou on who or what is pedophilia when y'all can't even get a hold of your own pedophilia problem. So shut the hell up and sit down
> 
> I'm 6'2, 102 kg.
> 
> Say it to my face next time.

Yeah, and?

**Comment by yagihime** [comment]
![406](https://user-images.githubusercontent.com/126365958/229642706-5550d29c-737c-4f2a-814e-314a4e062b8f.jpg)
thanks, i had to dig up 15 years of dust off this

**Comment by Oman395** [comment]
Hey alex, my discord is in the history, if you want to have a convo there

**Comment by logicP** [comment]
> > > > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > > > May I go on?
> > > > > 
> > > > > 
> > > > > Ay man go fuck yourself alright?
> > > > 
> > > > 
> > > > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > > > May I go on?
> > > > > 
> > > > > 
> > > > > Elon Musk is currently sitting very cosily in your small little rat brain, and he's not paying you any rent too.
> > > > 
> > > > 
> > > > Says the same person who's absolutely obsessed with being a troll and getting pissed off at someone calling you out. Also, you fuck yourself you transphobic little tool. You have absolutely no right to claim to be holier than thou on who or what is pedophilia when y'all can't even get a hold of your own pedophilia problem. So shut the hell up and sit down
> > > 
> > > 
> > > I'm 6'2, 102 kg.
> > > Say it to my face next time.
> > 
> > 
> > Yeah, and?
> 
> Fuck.
> 
> You.
> 
> Cunt.

lol
lmao even

**Comment by User0110001100** [comment]
fuck all of you

**Comment by Funtimes909** [comment]
> > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > May I go on?
> > > 
> > > 
> > > Ay man go fuck yourself alright?
> > 
> > 
> > > > "Drag show story time is pedophilia" says the same group of people where pedophilia is a recurring problem
> > > > "You're forcing your degeneracy onto kids" says the same group of people who praise children's beauty pageants in the same breath
> > > > "You're pushing the woke agenda", brought to you by the party of people that want to legalize child brides and child labor
> > > > May I go on?
> > > 
> > > 
> > > Elon Musk is currently sitting very cosily in your small little rat brain, and he's not paying you any rent too.
> > 
> > 
> > Says the same person who's absolutely obsessed with being a troll and getting pissed off at someone calling you out. Also, you fuck yourself you transphobic little tool. You have absolutely no right to claim to be holier than thou on who or what is pedophilia when y'all can't even get a hold of your own pedophilia problem. So shut the hell up and sit down
> 
> I'm 6'2, 102 kg.
> 
> Say it to my face next time.
> 
> I'll make you wish you were the blue bird.


fatass lol, explains your fetish for sitting behind a computer being a little piece of shit

**Comment by Funtimes909** [comment]
Stop whining about a women getting assaulted if you're just going to go and threaten violence to everyone who makes you bitch and cry for mommy because the meanies online are being hurtful 🤓🤓🤓🤓😭😭😭😭😭😭😭😭
